src/org/robockets/steamworks/commands/KillShooter.java

```diff
@@ -0,0 +1,35 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.robockets.steamworks.Robot;
+
+/**
+ * @author Jake Backer
+ */
+public class KillShooter extends Command {
```

Comment:
Let's try to test not using this command first. If WPILib's `end()` method is sufficient, then we should go with than.